deploy/charts/klicker-uzh-v2/values.yaml

```
@@ -311,5 +311,46 @@ lti:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 4
+    targetCPUUtilizationPercentage: 70
+    targetMemoryUtilizationPercentage: 80
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)